src/components/Checkbox/index.tsx

```diff
@@ -199,6 +209,7 @@ const Checkbox = forwardRef(
       autoFocus = false,
       className,
       'data-visibility': dataVisibility,
+      'aria-label': ariaLabel,
```

Comment:
This should probably be an XOR children | aria-label no ?

Reply:
Done